config/routes/liip_imagine.yaml

```diff
@@ -0,0 +1,2 @@
+_liip_imagine:
```

Comment:
Why the leading underscore?

```suggestion
liip_imagine:
```

Reply:
Don't think we should change, it' auto-generated by the bundle and works.